src/assisted_test_infra/test_infra/controllers/node_controllers/terraform_controller.py

```diff
@@ -282,15 +281,7 @@ def get_all_provisioning_addresses(self) -> List[str]:
         return addresses
 
     def set_dns(self, api_vip: str, ingress_vip: str) -> None:
-        base_domain = self._entity_config.base_dns_domain
-        fname = f"/etc/NetworkManager/dnsmasq.d/openshift-{self._entity_name}.conf"
-        contents = dedent(
-            f"""
-                    address=/api.{self._entity_name}.{base_domain}/{api_vip}
-                    address=/.apps.{self._entity_name}.{base_domain}/{ingress_vip}
-                    """
-        )
-        self.tf.change_variables({"dns_forwarding_file": contents, "dns_forwarding_file_name": fname})
+        utils.add_dns_record(self._entity_name, self._entity_config.base_dns_domain, api_vip, ingress_vip)
```

Comment:
I liked the named-parameters style you've made down below:
```suggestion
        utils.add_dns_record(
            cluster_name=self._entity_name,
            base_dns_domain=self._entity_config.base_dns_domain,
            api_vip=api_vip,
            ingress_vip=ingress_vip,
        )
```